appinventor/components/src/edu/mit/appinventor/ai/posenet/PosenetExtension.java

```diff
@@ -444,6 +461,16 @@ public String UseCamera() {
     return cameraMode;
   }
 
+  @SimpleProperty (description = "Configure video width.")
+  public void SetVideoWidth(int width) {
```

Comment:
We don't typically include `Set` as a keyword in property names. You should rename this to `VideoWidth`.